src/pages/Homepage.tsx

```diff
@@ -36,6 +38,10 @@ const Homepage: React.FC<RouteComponentProps> = (props) => {
           <Typography className="Button-Text">Instructions</Typography>
         </Button>
       </div>
+      <Instructions
```

Comment:
You can make this a self closing tag if you'd like
<Instructions prop1={} prop2={} />